android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,6 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="dev.google.cronet">
+  <!-- Flutter Boilerplate Modifications: Internet & Netwrok Permissions-->
```

Comment:
This is the another modified file that I mis-"copy pasted" with another manifest file's path.